src/simulations/collect.jl

```diff
@@ -53,6 +63,29 @@ two `DataFrame`s, one for agent-level data and one for model-level data.
 
 By default both keywords are `nothing`, i.e. nothing is collected/aggregated.
 
+## Mixed-Models
+
+  For mixed-models, the `adata` keyword has some additional options & properties.
+  An additional column `agent_type` will be placed in the output
+  dataframe.
+
+  In the case that data is needed for one agent type that does not exist
+  in a second agent type, `missing` values will be added to the dataframe.
+
+  **Warning:** Since this option is inherently type unstable, try to avoid this
+  in a performance critical situation. Performance will depend on specifics, but benchmarks
+  see a 10x slowdown when using `Symbol` matching and 26x for `Function` properties.
```

Comment:
My only comment actually is that I'd try to keep the docstring as lean as possible. "Performance will depend on specifics, but benchmarks see a 10x slowdown when using `Symbol` matching and 26x for `Function` properties." is something I'd remove.